docs/docusaurus/docs/core/customize_expectations/_examples/define_a_source_to_target_expectation.py

```diff
@@ -0,0 +1,118 @@
+"""
+This is an example script for how to define a source-to-target Expectation.
+
+To test, run:
+pytest --docs-tests -k "docs_example_define_a_source_to_target_expectation" tests/integration/test_script_runner.py
+"""
+
+
+def set_up_context_for_example(context):
+    # Create the Data Source
+    connection_string = "sqlite:///data/yellow_tripdata.db"
+    data_source_name = "my_sql_data_source"
+    data_source = context.data_sources.add_sqlite(
+        name=data_source_name, connection_string=connection_string
+    )
+    assert data_source.name == data_source_name
```

Comment:
Why these asserts on name? Feels like we should definitely be able to trust that data source is being created correctly here.

Reply:
I don't know. I based this new code sample off of the similar custom SQL Expectation code sample and [copied this line from there](https://github.com/great-expectations/great_expectations/blob/develop/docs/docusaurus/docs/core/customize_expectations/_examples/use_sql_to_define_a_custom_expectation.py#L16) 

If I should remove this assert / change it / replace it with different more meaningful assertations, please leave committable suggestions with the preferred code.